Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "ultra-sound-api"
+version = "0.1.0"
+edition = "2021"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[dependencies]
+axum = "0.6.12"
+serde = { version = "1.0.159", features = ["derive"] }
+sqlx = { version = "0.6.3", features = [
+  "json",
+  "offline",
+  "postgres",
+  "runtime-tokio-native-tls",
+] }
+tokio = { version = "1.27.0", features = ["full"] }
+tracing = "0.1.37"
+tracing-subscriber = "0.3.16"
+
+[dev-dependencies]
```

Comment:
How come these are only dev-dependencies? Are they only used in tests?

Reply:
Yes they are only used in these tests:
https://github.com/ultrasoundmoney/ultra-sound-api/pull/1/files#diff-0b979b6de560b29c1d97336878db56179224aa21d96fe099630c6628479ed020R24